test/getCommands.spec.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path');
+const Module = require('module');
+const expect = require('chai').expect;
+const getCommands = require('../src/getCommands');
+const findPlugins = require('../src/findPlugins');
+const mockRequire = require('mock-require');
+const mockFs = require('mock-fs');
+
+const commands = getCommands();
+const filesPath = path.join(__dirname, 'fixtures', 'files');
+const testPlugin = findPlugins(filesPath);
+const testPluginPath = path.join(__dirname, '..', 'node_modules', 'rnpm-plugin-test');
+const singlePlugin = require(path.join(filesPath, 'plugin'));
+const multiPlugin = require(path.join(filesPath, 'plugins'));
+
+describe('getCommands', () => {
+
+  before(() => {
+    mockRequire(path.join(__dirname, '..'), {});
+    mockRequire(
+      path.join(__dirname, '..', 'package.json'),
+      require(path.join(filesPath, 'package'))
+    );
```

Comment:
It's okey, but can't we just use `mock-fs` in that case? Because it's our test_suite, `__dirname/.../` equals `process.cwd()` anyway. That way, we don't have `mock-require` dependency at all.

Something like:
```js
mockFs({
   'package.json': require(path.join(filesPath, 'package'))
});
```
should be enough.